tests/conftest.py

```diff
@@ -168,19 +168,75 @@ def pytest_addoption(parser):
         ),
     )
     test_selection_group.addoption(
-        "--proxy",
-        "--proxy-tests",
-        dest="proxy",
-        action="store_true",
-        default=False,
-        help="Run proxy tests (DEPRECATED)",
+        "--no-fast",
+        "--no-fast-tests",
+        dest="fast",
+        action="store_false",
+        default=True,
+        help=(
+            "Run salt-fast tests. These fast tests that test States and Modules! "
+            "Default: True"
+        ),
```

Comment:
```suggestion
        "--no-fast",
        "--no-fast-tests",
        dest="fast",
        action="store_false",
        default=True,
        help=(
            "Run salt-fast tests. These fast tests that test States and Modules! "
            "Default: %{default)s"
        ),
```
A `--no-*` flag that runs tests as opposed to not running them? Is the help message wrong?